apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -192,7 +191,6 @@ const BaseStoryTemplate = (props: ModalProps) : TemplateResult => {
         <pie-button @click=${toggleModal}>Toggle Modal</pie-button>
         <pie-modal
             .aria="${aria}"
-            dir="${dir || nothing}"
```

Comment:
@raoufswe I plan to raise an immediate follow-up PR to update the component generator to no longer add the RTL component props stuff (as we won't need it if this change goes in).

I just want to keep this PR from getting out of hand 😄